### DISCUSSION TITLE
[✨] How to use Qwik with @vitejs/plugin-legacy to load legacy bundles?

### DISCUSSION THREAD

**Discussion by khromov** [QwikDev/qwik-evolution#123] [Proposals For Qwik]
### Is your feature request related to a problem?

I'd like to add `@vitejs/plugin-legacy` to my Qwik project. While adding `@vitejs/plugin-legacy` is possible and legacy bundles are being built, the code to load them is not correctly injected into the HTML on the SSR side. It's already well described [in this older issue](https://github.com/BuilderIO/qwik/issues/2785#issuecomment-1413478816), where it's said that "you can inject scripts in the header", but I don't understand how we can inject the required extra JS required to load legacy bundles, is it possible to add documentation for this?

### Describe the solution you'd like

A guide that shows how to integrate it properly so legacy chunks are referenced.

### Describe alternatives you've considered

Not supporting older browsers.

### Additional context

-

**Comment by mkatenolan** [comment]
Is there any update on this issue? I am facing the same problem, thanks!

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
